### ISSUE TITLE
rivergis problem with importing HECRAS

### ISSUE THREAD

**Issue by civilboy90** [erpas/rivergis#1]
although i've created my mesh and seen it in canvas with no problem, i couldn't import to HECRAS. i would be glad for any suggestion

**Comment by erpas** [comment]
Does RiverGIS create any HEC-RAS geometry file (*.g**)? How does it look like?

**Comment by civilboy90** [comment]
yes, it was created. i targeted one of my previously createg hecras 'g' files and when i opened it in hecras, the changes weren't reflected. then i tried to target a new geometry file but i see no data once i open it in hecras. File size is 444KB but no data. i tried to set schematic to an extend calculated thanks to QGIS canvas, but still can not see the result.

**Comment by erpas** [comment]
It is hard to say if I connot see the file. Can you post the geometry somewhere? Unfortunately, it is not possible hero on github to attach a text files.

**Comment by erpas** [comment]
Seems that Rivergis doesn't generate proper breaklines in your system, ie. your geometry data are all fine but they are are written to a geometry file as a single line. Can you give me some details about your system and environment (localization)?
